dev/.rat-excludes

```diff
@@ -99,3 +99,4 @@ spark-deps-.*
 .*tsv
 org.apache.spark.scheduler.ExternalClusterManager
 .*\.sql
+org.apache.spark.deploy.yarn.security.ServiceCredentialProvider
```

Comment:
I followed the convention of other services like `DataSource` which also needs to be excluded from rat check.